alfasim_sdk/hook_specs.py

```diff
@@ -208,6 +208,94 @@ def calculate_entrained_liquid_fraction(
     """
 
 
+def initialize_state_variables_calculator(
+    ctx: "void*",
+    P: "void*",
+    T: "void*",
+    T_mix: "void*",
+    n_control_volumes: "int",
+    n_layers: "int"
+) -> 'int':
+    """
+    Hook for the state variables calculator initialization.
+    To define a function that'll calculate the state variables for a given phase on ALFAsim,
+    two main steps must be performed:
+
+    1) In the plugin python configuration file, define the
+       alfasim_get_phase_properties_calculated_from_plugin function to set which phases
+       the current plugin is able to calculate state variables.
+
+    2) The plugin must, then, implement three hooks:
+    - HOOK_INITIALIZE_STATE_VARIABLE_CALCULATOR
+    - HOOK_CALCULATE_STATE_VARIABLE
```

Comment:
How about `HOOK_CALCULATE_PHASE_PAIR_STATE_VARIABLE`? Should it be listed here?

Reply:
yes